Dimensions/Dimensions.csproj

```diff
@@ -59,4 +56,4 @@
     <Compile Include="DataHandlers.cs" />
   </ItemGroup>
   <Import Project="$(MSBuildBinPath)\Microsoft.CSharp.targets" />
-</Project>
+</Project>
```

Comment:
For the references, you need to update the ones in the Refs folder instead of one in a path outside the project.